### ISSUE TITLE
Weird behavior of Sound Blaster in game Olimpiadas 92 - Atletismo

### ISSUE THREAD

**Issue by SmilingSpectre** [dosbox-staging/dosbox-staging#3859]
### Are you using the latest Dosbox-Staging Version?

- [X] I have checked releases and am using the latest release.

### What Operating System are you using?

Windows 10

### Have you checked that no other similar issue already exists?

- [X] I have searched and not found similar issues.

### A clear and concise description of what the bug is.

For the last half-year I encounter three games where SB behaves counter-intuitive, and the very similar way: the music is seemingly absent, but if you wait 10-20 seconds, it suddenly appears from nowhere. This effect doesn't affect FX the same way. It hardly can be deliberately made so, and three examples calls for more than silly coincidence (but still can be it).

The first one was an obscure Korean scroller-shooter I don't remember now, and I attributed the problem to the bad rip, because sound never played good there - it plays several notes in a bad tempo, then disappears for a random interval again.

The second one is Oxyd Magnum! Gold. It's harder to explain, as game was reissued several times, and with the same weird behavior as the Korean shooter - only in its case FX is utterly broken, and stops working after a very short time - and "music" then stops too.

And the third one is mentioned Olimpiad, which is remarkable in that aspect that FX is always present, and music works as expected - but it starts only ten seconds later than you expect it to work. There are online videos which demonstrate sounds right from the start of the game, so maybe real SB behave differently - but on the other hand, it also can be edited video.

Anyway, can you check the game and try to found why SB works as it works?

Additional issue of this game is appeared when I followed the hint from TDC and tried to use the different CT-VOICE drivers. I wasn't able to get music with any driver of versions higher than 1.xx. But If I try to use v3.07 from the Vogons pack, intended to SB16, with blaster=sb16, Staging crashed as soon as I press Ctrl, Alt or Win key with premortem message "Error: Rotate mode not supported". Same message appears in Daum fork, but it doesn't crash it.

Here are both mentioned files: Game with CT-VOICE which plays music, and CT-VOICE driver which crash Staging in this game.
[CT-VOICE.ZIP](https://github.com/user-attachments/files/16513772/CT-VOICE.ZIP)
[Olimpiadas 92 - Atletismo (1992).zip](https://github.com/user-attachments/files/16513777/Olimpiadas.92.-.Atletismo.1992.zip)

### Steps to reproduce the behaviour.

Explain how to reproduce
1. 
2.
3.

### Code of Conduct & Contributing Guidelines

- [X] Yes, I agree.

**Comment by johnnovak** [comment]
Thanks @SmilingSpectre. I assume you tried the games in original DOSBox, DOSBox SVN, Daum, ECE, and X, but all of them are affected by the issues?

Also, have you tried PCem or 86box? That would give us another data point if using those resolve the issues. I could perhaphs try in on hardware, but I only have an ESS board and an AWE32 in my MMX PC, and this game must expect an old SB1 or SB2 board.

**Comment by SmilingSpectre** [comment]
Yes, @johnnovak , I tried all DOSBoxes I have installed for checking delayed sound: vanilla 0.74.3, Daum, X (v.2023.10.6) and ECE.

I didn't try to check "rotate" issue, as I supposed it's some exclusive Staging bug. And was surprised when seen it in Daum too. I didn't check other versions for this issue.

**Comment by johnnovak** [comment]
> I didn't try to check "rotate" issue, as I supposed it's some exclusive Staging bug. And was surprised when seen it in Daum too. I didn't check other versions for this issue.

That's a legacy thing everybody inherited from the original DOSBox code. It's just some unimplemented stuff. Original DOSBox and thus Staging exits when it goes into that code path (that's the 0.74 behaviour), but apparently Daum or ECE just logs and error.

**Comment by johnnovak** [comment]
Added "low priority" as this is a bottom of the barrel game. We'll fix it when we'll have nothing better to do 😉
